federatedscope/autotune/utils.py

```diff
@@ -166,11 +256,32 @@ def eval_in_fs(cfg, config, budget, client_cfgs=None):
     data, modified_config = get_data(config=trial_cfg.clone())
     trial_cfg.merge_from_other_cfg(modified_config)
     trial_cfg.freeze()
-    Fed_runner = get_runner(data=data,
+    fed_runner = get_runner(data=data,
                             server_class=get_server_cls(trial_cfg),
                             client_class=get_client_cls(trial_cfg),
                             config=trial_cfg.clone(),
-                            client_config=client_cfgs)
-    results = Fed_runner.run()
+                            client_configs=client_cfgs)
+    results = fed_runner.run()
+
+    return results
+
+
+def config_bool2int(config):
+    # TODO: refactor bool/str to int
+    import copy
+    new_dict = copy.deepcopy(config)
+    for key, value in new_dict.items():
+        if isinstance(new_dict[key], bool):
+            new_dict[key] = int(value)
+    return new_dict
+
+
+def log2wandb(trial, config, results, trial_cfg):
+    import wandb
```

Comment:
As I have no idea about the overhead of repeatedly importing a module, I am wondering is it appropriate to do this? Have you benchmarked such a behavior? Here is a related thread: https://stackoverflow.com/questions/128478/should-import-statements-always-be-at-the-top-of-a-module